src/background/background-process.js

```diff
@@ -75,7 +75,7 @@ export default class BackgroundProcess {
   }
 
   updateIcon (active, tabId) {
-    let params = { path: active ? this.browserIcons.active : this.browserIcons.nonactive }
+    const params = { path: active ? this.browserIcons.active : this.browserIcons.nonactive }
```

Comment:
I believe that we should use `let` here - as we could change properties in the following line.

Reply:
ok, will override the linter